rfcs/text/0015-create-file-elf.md

```diff
@@ -86,10 +88,87 @@ Stage 1: Provide a high-level description of example sources of data. This does
 **Stage 2**
 
 ### Real world examples
-
 <!--
 Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
+https[://]c4a08b8258a848efb518dcea18a1d11e[.]us-central1[.]gcp[.]cloud[.]es[.]io/goto/22bceecfba7e40710e8c5ddefa853bc8
```

Comment:
nit: Do we wanna leave this in the comment?

Reply:
Fix't